src/controllers/recipesController.js

```diff
@@ -0,0 +1,84 @@
+const BaseController = require('./baseController')
+
+class RecipesController extends BaseController {
+  constructor(a) {
+    super(a)
+  }
+
+  async index(req, res) {
+    const recipes = await this.models.recipes.findAll({
+      include: this.models.users,
+    })
+
+    var output = new Set()
+    recipes.forEach((item) => {
+      output.add(item.type)
+    })
+
+    const types = output
+
+    res.render('pages/recipes/index.html.ejs', {
+      recipes,
+      types,
+      context: await this.viewContext(req),
+    })
+  }
+
+  async show(req, res) {
+    const recipe = await this.models.recipes.findOne({
+      where: {
+        id: req.params.id,
+      },
+
+      include: [
+        this.models.users,
+        this.models.recipe_ratings,
+        {
+          model: this.models.comments,
+          include: this.models.users,
+        },
+      ],
+      order: [[this.models.comments, 'createdA', 'DESC']],
```

Comment:
Stulpelio pavadinimas `createdAt`:
```suggestion
      order: [[this.models.comments, 'createdAt', 'DESC']],
```